src/LogViewer/index.js

```diff
@@ -275,23 +277,29 @@ export default class LogViewer extends React.Component {
   }
 
   jumpToLine(lineNumber) {
-    const { minLineHeight, offset } = this.state;
+    const { offset, lineHeight } = this.state;
 
-    window.scrollTo(0, (lineNumber - offset) * minLineHeight + minLineHeight);
+    window.scrollTo(0, (lineNumber - offset) * lineHeight + 35);
```

Comment:
`35` is the current `margin-top` css value of `#log-container`. Not really sure if I should keep it like that or do `document.getElementById("log-container")` and fetch its value. I'm thinking if I do that, it will add delay.